Fix dev-infra zone.js substitution

wagnermaciel opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

wagnermaciel committed:
fix(dev-infra): fix broken zone.js substitution for dev-infra:npm_package

* fix substitution that was broken by PR #36540 to match
  the new import path

devversion reviewed (approved):
LGTM. We probably need to update the target path when there is as new zone.js release. We should then import from `zone.js/bundles/zone.umd.js` I guess.

wagnermaciel commented:
I'm gonna re-test this tomorrow. I am afraid I might have tested against a local / cached version of dev-infra that I modified manually instead of the one build off of this branch. For some reason I need to run `yarn bazel clean --expunge` for each new build I want to test

wagnermaciel committed:
fix(dev-infra): add missing BUILD file to dev-infra/bazel:files

* Without this BUILD file we were seeing errors about the reference to
  expand_template.bzl in ng_rollup_bundle.bzl because dev-infra/bazel
  was not considered a package.

devversion reviewed (approved):
LGTM +💯
